src/lib/builder/board-builder.ts

```diff
@@ -3,13 +3,28 @@ import {
   createGroupBuilder,
   type GroupBuilder,
 } from "./group-builder"
-import * as Type from "lib/types"
+import type * as Type from "lib/types"
 import * as CB from "./component-builder"
-import { TraceBuilder } from "./trace-builder"
+import type { TraceBuilder } from "./trace-builder"
 import { createTraceBuilder } from "./trace-builder"
-import { ProjectBuilder } from "./project-builder"
+import type { ProjectBuilder } from "./project-builder"
 import { createTraceHintBuilder } from "./trace-hint-builder"
 import { createNetBuilder } from "./net-builder/net-builder"
+import type { BoardProps as OriginalBoardProps } from "@tscircuit/props"
+
+interface ExtendedBoardProps extends OriginalBoardProps {
+  board_thickness?: number | string
+  center?: Type.Point
+  pcbX?: number | string
+  pcbY?: number | string
+}
```

Comment:
```suggestion
```

Just use `BoardProps` from `@tscircuit/props`. You'll need to update it to the latest version